### PR TITLE
IOD-194 Adding Health Check

### PR DESCRIPTION
**Pull request by akroon3r**
# Description

This PR includes the following proposed change(s):

IOD-194

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Refactoring / Documentation
- [ ] Version change

if your change is a breaking change, please add `breaking change` label to this PR

## How Has This Been Tested?

Please describe the tests that you ran to verify your changes.

## Does the change impact or break the Docker build?

- [ ] Yes
- [x] No

If Yes: Has Docker been updated accordingly?

- [ ] Yes
- [x] No

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] I have made corresponding changes to the documentation
- [x] New and existing unit tests pass locally with my changes

### CODE REVIEW

**Review by SDToews — COMMENTED** [review]
as discussed

File: package.json [hunk 1]
```diff
@@ -21,11 +21,13 @@
     "dev": "npm run start:dev"
   },
   "dependencies": {
+    "@nestjs/axios": "^0.0.7",
```
Comment (akroon3r) [inline package.json:24]: remove carat

File: package.json [hunk 2]
```diff
@@ -21,11 +21,13 @@
...
     "@nestjs/common": "8.3.0",
     "@nestjs/config": "1.2.0",
     "@nestjs/core": "8.3.0",
     "@nestjs/platform-express": "8.3.0",
     "@nestjs/swagger": "5.2.0",
+    "@nestjs/terminus": "^8.0.6",
```
Comment (akroon3r) [inline package.json:30]: remove carat

File: src/health/health.controller.ts [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+  constructor(
+    private health: HealthCheckService,
+    private http: HttpHealthIndicator,
+  ) {}
+
+  @Get()
+  @HealthCheck()
+  check() {
+    return this.health.check([
+      () => this.http.pingCheck('nestjs-docs', 'https://docs.nestjs.com'),
```
Comment (akroon3r) [inline src/health/health.controller.ts:19]: test hitting 8.8.8.8 instead

**Comment by akroon3r** [comment]
@SDToews issues are resolved and ready for review

**Review by SDToews — APPROVED** [review]

**Pull request merged**